src/parseTools.js

```diff
@@ -1546,6 +1546,9 @@ function makeRetainedCompilerSettings() {
   return ret;
 }
 
+// Page size of the virtual filesystem.
+var FS_PAGE_SIZE = 16384;
```

Comment:
Seems like it s not just the filesystem but also mmap that uses this.

How about calling this `USER_PAGE_SIZE`?   Or mabye something more meaningfull.  I'm not sure that the purpose of it is exactly.

Reply:
Happy to call it `USER_PAGE_SIZE`, but it is used in the filesystem layer and nowhere else, so I think `FS_` makes sense? (the old name for it was just `PAGE_SIZE` which was ambiguous)

Reply:
I looks like it used by `getpagesize` and by `mmap` .. I guess you could argue that mmap is FS related but not solely (MAP_ANON etc).

Reply:
Hmm, yeah. We don't support MAP_ANON though, so it's purely FS for now.

I think FS is better than USER, as USER is more ambiguous. How strongly do you feel about this?

Reply:
I don't feel super strongly.   getpagesize is nothing to do with filesystem though.

How about REPORTED_PAGE_SIZE?  Or POSIX_PAGE_SIZE?

Reply:
lgtm either way

Reply:
I like `POSIX_PAGE_SIZE`, switched to that. Added a clarifying comment too.